src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -339,8 +360,8 @@ void SetParameter()
       Aux_Message( stdout, "  central angular frequency Omega_0 (in rad/s)        = %13.7e\n", CCSN_CC_Rot_Omega0       ); }
       if ( CCSN_CC_Rot == 2 )
       Aux_Message( stdout, "  multiplication factor for rotational profile        = %13.7e\n", CCSN_CC_Rot_Fac          );
-      Aux_Message( stdout, "  minimum angular resolution                          = %13.7e\n", CCSN_Min_AngRes          );
-      Aux_Message( stdout, "  maximum angular resolution                          = %13.7e\n", CCSN_Max_AngRes          );
+      Aux_Message( stdout, "  minimum angular resolution                          = %13.7e\n", CCSN_AngRes_Min/DEG2RAD  );
+      Aux_Message( stdout, "  maximum angular resolution                          = %13.7e\n", CCSN_AngRes_Max/DEG2RAD  );
```

Comment:
it is fine to print these values in degrees or radians, but it would be better to add the unit in the description.

Reply:
I think it is easier to see degree, so I added unit descriptions